### PR TITLE
Archive trillinos and pytrillinos

### PR DESCRIPTION
**Pull request by hmaarrfk**
They seem rather abbandoned

- https://github.com/conda-forge/trilinos-feedstock/pulls
- https://github.com/conda-forge/pytrilinos-feedstock/pulls

## Guidelines for marking packages as broken:

* We prefer to patch the repo data (see [here](https://github.com/conda-forge/conda-forge-repodata-patches-feedstock))
  instead of marking packages as broken. This alternative workflow makes environments more reproducible.
* Packages with requirements/metadata that are too strict but otherwise work are
  not technically broken and should not be marked as such.
* Packages with missing metadata can be marked as broken on a temporary basis
  but should be patched in the repo data and be marked unbroken later.
* In some cases where the number of users of a package is small or it is used by
  the maintainers only, we can allow packages to be marked broken more liberally.
* We (`conda-forge/core`) try to make a decision on these requests within 24 hours.

What will happen when a package is marked broken?

* Our bots will add the `broken` label to the package. The `main` label will remain on the package and this is normal.
* Our bots will rebuild our repodata patches to remove this package from the repodata.
* In a few hours after the `anaconda.org` CDN picks up the new patches, you will no longer be able to install the package from the `main` channel.

## Checklist:

* [ ] I want to mark a package as broken (or not broken):
  * [ ] Make sure your package is in the right spot (`broken/*` for adding the
    `broken` label, `not_broken/*` for removing the `broken` label, or `token_reset/*`
    for token resets)
  * [ ] Added a description of the problem with the package in the PR description.
  * [ ] Pinged the team for the package for their input.

* [ ] I want to archive a feedstock:
  * [ ] Pinged the team for that feedstock for their input.
  * [ ] Make sure you have opened an issue on the feedstock explaining why it was archived.
  * [ ] Linked that issue in this PR description.
  * [ ] Added links to any other relevant issues/PRs in the PR description.

### CODE REVIEW

File: archive/trillinos.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+trillinos
```
Comment (beckermr) [inline archive/trillinos.txt:1]:
```suggestion
trilinos
```

File: archive/pytrillinos.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+pytrillinos
```
Comment (beckermr) [inline archive/pytrillinos.txt:1]:
```suggestion
pytrilinos
```

**Review by beckermr — COMMENTED** [review]
Wrong names.

**Comment by hmaarrfk** [comment]
Well that was embarrassing

**Comment by beckermr** [comment]
Regarding the actual PR, we don't typically archive abandoned feedstocks. Is there a specific reason we should here?

**Comment by hmaarrfk** [comment]
it just really holds up a few tough migrations making it hard to understand how far they actually got.

**Comment by beckermr** [comment]
Hmmmmm ok. I wonder if there is a way we can get the bot to skip them besides archiving.

**Comment by beckermr** [comment]
We could add a feature that lets people have the bot skip their feedstock. I'm not sure how I feel about this.

**Comment by hmaarrfk** [comment]
Hmm ok, lets discuss ideas: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/migrations/hdf51122.yaml

**Comment by hmaarrfk** [comment]
Opps: ---- > https://github.com/conda-forge/conda-forge.github.io/issues/1724

**Pull request closed without merging**